### DISCUSSION TITLE
Selectolax

### DISCUSSION THREAD

**Discussion by baseplate-admin** [flet-dev/flet#3739] [Packages] [answered]
Add [selectolax](https://pypi.org/project/selectolax/) for flet

Source :
- [Github](https://github.com/rushter/selectolax)
- [PyPi](https://pypi.org/project/selectolax/)

Notes:
- Uses Cython to bind with C++

**Comment by ndonkoHenri** [comment, marked as answer]
Selectolax 0.4.10 has been published for both Android and iOS: https://pypi.flet.dev/selectolax
Give it a try and let us know how it worked.
